csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -816,6 +875,16 @@ static Settings()
             /// </summary>
             public bool FormatEnumsAsIntegers { get; }
 
+            /// <summary>
+            /// Indentation string, used for formatting.
+            /// </summary>
+            public string Indent { get; }
```

Comment:
This is very subjective, but to me "indent" sounds like a Boolean - whether or not something is indented. Whereas "indentation" is "the amount by which the text is indented". So I'd rename this to `Indentation`, along with the method and constructor parameters.
(Aside from anything else, that's more consistent - `WithIndentation(...)` and `Indentation` match...)

Reply:
It's not the right name indeed. Fixed this :-)